monitoring/snippets/v3/cloud-client/snippets.py

```diff
@@ -48,7 +58,12 @@ def create_metric_descriptor(project_id):
     # [END monitoring_create_metric]
 
 
-def delete_metric_descriptor(descriptor_name):
+def delete_metric_descriptor(descriptor_name: str) -> None:
```

Comment:
Can you please redo the sample to return a value and use it in the test instead of output parsing?

Reply:
The call to "delete" method does not return value. Without reverse-engineering the client library I would guess it throws exception if the operation fails. What would you suggest to be the code for handling failure?